### DISCUSSION TITLE
When using css module, styles does not work or doesn't apply!?

### DISCUSSION THREAD

**Discussion by MahirAlam** [mantinedev/mantine#5440] [Q&A]
### When I use a style on container, It Just Doesn't work!

**It shows the Style Name But doesn't show the styles or apply it!**
You can see It Here
![Screenshot 2023-12-18 040432](https://github.com/mantinedev/mantine/assets/103423298/f588fa39-4bda-4e4e-b8e9-cf828d857320)
![Screenshot 2023-12-18 040506](https://github.com/mantinedev/mantine/assets/103423298/8c9b9609-ca36-4e1e-9111-bf30ecd2c95c)
![Screenshot 2023-12-18 040532](https://github.com/mantinedev/mantine/assets/103423298/1b081697-44dd-403d-87ba-092a5fa287f4)

**Comment by rtivital** [comment]
It is impossible to tell where the issue is without any context. Reproduce it on codesandbox.

**Reply by MahirAlam** [reply]
I am sorry bro, I can't at this moment!

You can view my code for example!
GitHub URL: https://github.com/MahirAlam/test-run
